modules/client/src/client-common/interfaces/plugin.ts

```diff
@@ -203,7 +213,7 @@ export enum VoteProposalStep {
 
 export type VoteProposalStepValue =
   | { key: VoteProposalStep.VOTING; txHash: string }
-  | { key: VoteProposalStep.DONE; voteId: string };
+  | { key: VoteProposalStep.DONE };
```

Comment:
Don't we have the ID of the vote anymore?

Reply:
Same as before I couldnt find an event thrown in the contracts